.dotnet.azure/sdk/openai/Azure.AI.OpenAI/src/Custom/Common/Internal/InternalAzureContentFilterBlocklistResultDetail.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ClientModel.Primitives;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Azure.AI.OpenAI;
+
+[Experimental("AOAI001")][CodeGenModel("AzureContentFilterBlocklistResultDetail")]
+internal partial class InternalAzureContentFilterBlocklistResultDetail
+{
+    [CodeGenMember("Filtered")]
+    internal bool IsFiltered { get; set; }
```

Comment:
nit: Filtered

Reply:
Even better -- the file goes away and this goes back to `GeneratorStubs`!